.github/workflows/dockerimage-next.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       DWO_BUNDLE_IMG: quay.io/devfile/devworkspace-operator-bundle:next
       DWO_INDEX_IMG: quay.io/devfile/devworkspace-operator-index:next
       OPM_VERSION: v1.17.1
-      OPERATOR_SDK_VERSION: v1.7.2
+      OPERATOR_SDK_VERSION: v1.8.0
```

Comment:
You forced me to do some manual work which conflicts with che operator.
So, ideally it would be great to use bingo or something https://github.com/devfile/devworkspace-operator/pull/476 but it's not related to your PR.